src/metrics_server/connection_metrics.ts

```diff
@@ -114,6 +117,11 @@ export function isValidConnectionMetricsReport(
       return false;
     }
 
+    // Check that `tunnelTime` is a number larger than min tunnel time.
+    if (typeof userReport.tunnelTimeMs !== 'number' || userReport.tunnelTimeMs < MIN_TUNNEL_TIME) {
```

Comment:
Can you inline the zero here instead? It's more readable. Otherwise I need to lookup MIN_TUNNEL_TIME. Zero doesn't really deserve a constant, and it won't change.